src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -128,6 +130,41 @@ trait BackendCall {
     throw new NotImplementedError(s"resume() called on a non-resumable BackendCall: $this")
   }
 
+  def useCachedCall(cachedBackendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = {
+    throw new NotImplementedError(s"useCachedCall() is not implemented for BackendCall: $this")
+  }
+
+  /**
+   * Compute a hash that uniquely identifies this call
+   */
+  def hash: String = {
+    val orderedInputs = ListMap(locallyQualifiedInputs.toSeq.sortBy(_._1): _*)
+    val orderedOutputs = call.task.outputs.sortWith((l, r) => l.name > r.name)
+    val runtime = call.task.runtimeAttributes
+    val orderedRuntime = ListMap(
+      ("docker", runtime.docker.getOrElse("")),
```

Comment:
I'm not using the proper Docker hash yet, just the name of the image in the WDL file.  This is where we'd hook into Khalid's code